apps/avifenc.c

```diff
@@ -611,7 +611,7 @@ static avifBool avifInputReadImage(avifInput * input,
                                                             ignoreXMP,
                                                             allowChangingCicp,
                                                             ignoreGainMap,
-                                                            AVIF_DEFAULT_IMAGE_SIZE_LIMIT,
```

Comment:
Should it also be changed at line 585?

Reply:
Done. Thanks.

I found that `AVIF_DEFAULT_IMAGE_SIZE_LIMIT` is also used in two places in `avifgainmaputil`. I didn't look at them to see if either of them should be changed to `UINT32_MAX`.